installer/scripts/genRubyInstaller.sh

```diff
@@ -52,14 +52,20 @@ if [ -z "$SOURCE_DIR" -o ! -d $SOURCE_DIR ]; then
     exit 1
```

Comment:
#1 is already covered in existing code.  I did not see the need for #2 as in my scenario I used /opt/Microsoft/ path for getting the ruby changes from (which is covered in the second part of the how to doc) where you do not have to have the make ran already prior to running genRubyInstaller.sh.